invenio.cfg

```diff
@@ -868,36 +868,11 @@ MIRADOR_PREVIEW_EXTENSIONS = [
 """Mirador previewer allowed extensions."""
 
 MIRADOR_PREVIEW_CONFIG = {
-    "window": {
-        "allowClose": False,
-        "allowFullscreen": True,
-        "allowTopMenuButton": True,
-        "allowMaximize": False,
-        "allowWindowSideBar": True,
-        "defaultSidebarPanelHeight": 201,
-        "defaultSidebarPanelWidth": 235,
-        "defaultView": "xsingle",
-        "hideWindowTitle": True,
-        "sideBarOpen": False,
-        "switchCanvasOnSearch": False,
-        "panels": {
-            "info": True,
-            "attribution": False,
-            "canvas": False,
-            "annotations": False,
-            "search": False,
-            "layers": False,
-        },
-        "views": [
-            {"key": "single", "behaviors": ["individuals"]},
-            {"key": "book", "behaviors": ["paged"]},
-            {"key": "scroll", "behaviors": ["continuous"]},
-            {"key": "gallery"},
-        ],
-    },
+
```

Comment:
since I am updating the flags in the `window` var in the mirador dist in default config based on existence of annotations, I removed it from here, as it overrides default config. If we'd rather have it here, then I can instead update the config in `mirador.py` instead